src/fieldset/index.jsx

```diff
@@ -65,9 +65,13 @@ const fields = {
   declaration: props => <ApplicationConfirm { ...props } />,
   inputDate: props => <DateInput { ...props } onChange={value => props.onChange({ target: { value } })} />,
   textarea: props => <TextArea { ...props } autoExpand={true} />,
-  radioGroup: props => props.options.length > 1
-    ? <RadioGroup { ...props } />
-    : <SingleRadio { ...props } />,
+  radioGroup: props => {
+    if (props.options === undefined) {
```

Comment:
```suggestion
    if (!props.options) {
```

We probably want the same level of debugging for other failure states.